templates/default/service-uivm

```diff
@@ -66,7 +66,7 @@
     "hvm": "false",
     "apic": "true",
     "nx": "true",
-    "vkbd" : "true",
+    "vkb" : "true",
```

Comment:
Same as above, if we really want to rename this attribute we need a migration script.
Maybe it's best to just keep the old name?